pkg/aws/pca_test.go

```diff
@@ -325,8 +325,9 @@ func TestIdempotencyToken(t *testing.T) {
 					Namespace: "fake-namespace",
 				},
 			},
-			expected: "f331cbfd0cc6569f58c12c3dbb238a4f",
+			expected: "63e69830270b95081942a3d85034fdc97bb9f0", // Truncated SHA-256 hash
```

Comment:
Looks like this new token is 38 characters

Reply:
Done, sorry for the miscount, thanks for catching it